examples/queryWithParams.ts

```diff
@@ -9,12 +9,12 @@ import {
   flux,
   fluxDuration,
 } from '@influxdata/influxdb-client'
-import {url, token, org} from './env'
+import {url, token, org, bucket} from './env'
 
 const queryApi = new InfluxDB({url, token}).getQueryApi(org)
 const start = fluxDuration('-1m')
 const measurement = 'temperature'
-const fluxQuery = flux`from(bucket:"my-bucket") 
+const fluxQuery = flux`from(bucket:"${bucket || 'my-bucket'}") 
```

Comment:
I find it confusing that we imply that the user can or should configure the bucket, but then we ignore it in the code, especially in the browser query example. We could make it simpler for the user and default to their configured bucket instead of leaving it for them to figure out.

Reply:
@sranka Trying to clarify my earlier point...
I find it confusing that we imply that the user can or should configure the bucket, but then we ignore it in the code, especially in the browser query example. Isn't it simpler to prefer the user-supplied configuration and then default to "my-bucket"? That seems a better user experience than using all their config except for the bucket and leaving it for them to figure out why.

Reply:
@jstirnaman I think that I understand your point. Creating a flux query using a standard javascript template string (or string concatenation) IS NOT something that I want to show in the examples, it opens door to injection attacks. A simple string (without variables) is easier to understand for first-time users. 

I would rather improve the comments in `env.js` and `env-browser.js` to something like

```
InfluxDB bucket name is required to write the data, it is constant in most of the applications. The examples expect that the bucket name is "my-bucket".
```

or we should remove the bucket from the env files and use it locally in all the other places.